test/fuzz/lua/test_engine.lua

```diff
@@ -848,6 +1157,7 @@ local ops = {
         args = function(space)
             return random_tuple(space:format()), random_tuple_operations(space)
         end,
+        engines = {'memtx', 'vinyl'},
```

Comment:
Do we really need additional field if it is required in a single operation?
I propose to infer engine name using index and return from an operation function if engine is not supported:

```lua

tarantool> box.space.aa.index['pk'].space_id
---
- 512
...

tarantool> box.space[512].engine
---
- memtx
...

tarantool> 
```

What do you think?

Reply:
Okay, thanks! (Actually, I've added a comment on review an intermediate commit when didn't see the latest commit where more engines limitation were added.)